The Komodo side panes will select the incorrect widget (tabpanel) for the
selected tab, due to a XBL issue?
diff -r 4cfb6c61b137 toolkit/content/widgets/tabbox.xml
--- a/toolkit/content/widgets/tabbox.xml	Thu Feb 27 15:34:11 2014 +0100
+++ b/toolkit/content/widgets/tabbox.xml	Thu Mar 27 14:45:25 2014 -0700
@@ -674,7 +674,8 @@
         <setter>
           <![CDATA[
             var selectedIndex = -1;
-            for (var panel = val; panel != null; panel = panel.previousSibling)
+            // KOMODO - Our tabpanel panes unexpectedly run into a "children" previousSibling.
+            for (var panel = val; panel != null && panel.tagName != "children"; panel = panel.previousSibling) {
               ++selectedIndex;
             this.selectedIndex = selectedIndex;
             return val;
